app/src/main/res/layout-sw320dp/header.xml

```diff
@@ -2,67 +2,31 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="150dp"
-    android:background="@color/amaranth_red"
-    android:gravity="bottom">
-
-    <ImageView
-        android:id="@+id/imageView6"
-        android:layout_width="73dp"
-        android:layout_height="77dp"
-        android:layout_marginStart="16dp"
-        android:layout_marginBottom="16dp"
-        android:src="@drawable/run_man"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.8"
-        android:contentDescription="@string/header_imageViewProfile" />
+    android:layout_height="wrap_content"
+    android:background="@color/primary"
+    android:gravity="bottom"
+    android:paddingTop="30dp"
+    android:paddingBottom="10dp">
 
     <TextView
-        android:id="@+id/text_view_profile_name"
-        android:layout_width="wrap_content"
+        android:id="@+id/header_title"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginStart="8dp"
-        android:layout_marginTop="50dp"
-        android:text="@string/header_name"
-        android:textAlignment="viewStart"
-        android:textSize="20sp"
+        android:text="@string/app_name"
+        android:textSize="36sp"
         android:textStyle="bold"
-        app:layout_constraintStart_toEndOf="@+id/imageView6"
-        app:layout_constraintTop_toTopOf="parent">
-
-
-    </TextView>
-
-    <TextView
-        android:id="@+id/text_view_profile_phone"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="8dp"
-        android:text="@string/header_phone"
-        android:textAlignment="viewStart"
-        android:textSize="14sp"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toEndOf="@+id/imageView6"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.648">
-
-    </TextView>
+        app:fontFamily="@font/aclonica"
```

Comment:
Didn't we use the "armata" font in the beginning, why do you want to change this?

Reply:
Yes, but here it's only for the text "MOVEMEET" in the header and it's the font that we already used on the MainActivity

Reply:
Yes, but here it's only for the text "MOVEMEET" in the header and it's the font that we already used on the MainActivity